### ISSUE TITLE
Unknown property `allowTransparency` found (`react/no-unknown-property`)

### ISSUE THREAD

**Issue by frixaco** [jsx-eslint/eslint-plugin-react#3489]
Hi and thank you for the work on the plugin!

I'm getting this error when I try to lint the following code:
```tsx
import './App.css'

function App(): JSX.Element {
  return (
    <div className="App">
      <iframe allowTransparency />
    </div>
  )
}

export default App
```
Any ideas why this is happening? Looking at source code, it seems like `allowTransparency` is a valid property.

Reproduction: https://codesandbox.io/s/react-check-eslint-5utwqm?file=/src/App.tsx

eslint-plugin-react: 7.31.10
ESLint: v7.32.0
React: v18 and v16.13.0

**Comment by ljharb** [comment]
http://help.dottoro.com/ljjvodln.php suggests that this property was only ever supported by IE < 9. Does this attribute seem to have a different effect for you in modern browsers?

**Comment by frixaco** [comment]
Sorry for **very** late reply. And no, I did not notice any effects on Chrome browser.

Closing the issue. I can't reproduce it anymore (so it's either was an issue with my types or have been fixed).
